### ISSUE TITLE
Problem with versions of Vue 3.2.25 and later

### ISSUE THREAD

**Issue by rudolfaccenture** [chyj4747/vue3-golden-layout-virtualcomponent#6]
This line and the others in which this property is accessed as is throws an error in versions of Vue from 3.2.25 and onwards, because its value is an array.
Accessing the property as the first element of an array (so, basically just adding "[0]" after "component.glc") fixes the issue according to my testing.

https://github.com/chyj4747/vue3-golden-layout-virtualcomponent/blob/5ce17d3ec7a93914bd08ea56cfc455f95608812b/src/components/Glayout.vue#L207

**Comment by weigao-123** [comment]
Yes, recently I also tried Vue3.2.37 with Vite and has the same issue. More details can be found here #4
